hieradata/cluster/manke.yaml

```diff
@@ -161,7 +161,7 @@ profile::nm::connections:
       route1_options=table=2505
       route2=0.0.0.0/0,139.229.153.254
       route2_options=table=2505
-      routing-rule1=priority 100 from 193.229.153.64/26 table 2505
+      routing-rule1=priority 100 from 139.229.153.64/26 table 2505
```

Comment:
FYI - https://github.com/lsst-it/lsst-itconf/pull/751 will add spec coverage of this.

Reply:
you prefer to leave it with the typo?